easybuild/easyconfigs/b/blasr_libcpp/blasr_libcpp-20170426-intel-2017a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'blasr_libcpp'
+version = '20170426'
+commit = 'cd49f36'
+
+homepage = 'https://github.com/PacificBiosciences/blasr_libcpp'
+description = """Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for
+ analyzing PacBio sequences"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['https://github.com/PacificBiosciences/blasr_libcpp/archive/']
+sources = ['%s.tar.gz' % commit]
+
+dependencies = [
+    ('HDF5', '1.8.18', '-serial'),
+    ('pbbam', '20170508'),
+]
+
+unpack_options = "--strip-components=1"
+
+skipsteps = ['configure', 'install']
+buildininstalldir = True
+
+#prebuildopts = "NOPBBAM=1 NOHDF=1 ./configure.py && "
```

Comment:
Using `NOPBBAM=` and `NOHDF=` didn't do what I think it would.

I thought it meant not to build own versions of `ppbam` and `HDF(5)`, but it actually means disable the features that require those dependencies, which is not what we want.

So, I'll just clean this up, thanks for bringing it up!